src/ucp/wireup/wireup_cm.c

```diff
@@ -421,7 +443,13 @@ static unsigned ucp_ep_cm_disconnect_progress(void *arg)
         /* if the EP is not local connected, the EP has been flushed and CM lane is
          * disconnected, schedule close request completion and EP destroy */
         ucs_assert(ucp_ep->flags & UCP_EP_FLAG_CLOSE_REQ_VALID);
-        ucp_ep_local_disconnect_progress(ucp_ep_ext_gen(ucp_ep)->close_req.req);
+        close_req = ucp_ep_ext_gen(ucp_ep)->close_req.req;
+        if (UCS_PTR_IS_PTR(close_req)) {
```

Comment:
how can we have CLOSE_REQ_VALID but no close_req ptr?

Reply:
see https://github.com/openucx/ucx/pull/4445#discussion_r357607189

Reply:
see answer for it, also - if we failed to allocate request, it's enough to print error once. no need to print error also when we get disconnect event for this ep